catalogue/webapp/components/WorkSearchResults/WorkSearchResults.js

```diff
@@ -24,7 +24,7 @@ const WorkSearchResults = ({ works, worksRouteProps, apiProps }: Props) => {
   const { relevanceRating } = useContext(TogglesContext);
 
   return (
-    <div className={'grid'}>
+    <div className={'grid'} data-test-id="search-results">
```

Comment:
Having to add an ID just so the tests can identify this seems a bit weird.

Reply:
Yea, I would normally say the same, but it is [common practise](https://testing-library.com/docs/dom-testing-library/api-queries/#bytestid) on a lot of libraries.

It's be cause we rarely add anything unique to elements. For form elements etc this is easy, or for elements that have roles (a preferred way) - but for something like a div that shows when there is results, it's a last result.

We wouldn't want to add a normal ID, as that would imply that it is being used somewhere, which we aren't.

[We use it in a few other places too](https://github.com/wellcomecollection/wellcomecollection.org/search?q=data-test-id)